File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    s_list = s.split()
    
    min = int(s_list[0])
    max = int(s_list[0])
    for i in s_list :
        if min > int(i):
            min = int(i)
        if max < int(i):
            max = int(i)
    print(min, max)
    
    answer = str(min) + " " + str(max)
        
    return answer

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] s_list = s.split(" ");
        
        Integer min = Integer.parseInt(s_list[0]);
        Integer max = Integer.parseInt(s_list[0]);
        
        for(String c : s_list){
            Integer num = Integer.parseInt(c);
            if(num > max){
                max = num;
            }
            if (num<min){
                min = num; 
            }
        }
        answer = "" + min + " " + max; 
        
        return answer;
    }
}